mat64/list_test.go

```diff
@@ -230,10 +230,10 @@ func returnAs(a, t Matrix) Matrix {
 // of a.
 func retranspose(a, m Matrix) Matrix {
 	switch a.(type) {
-	case Transpose:
-		return Transpose{m}
 	case TransposeTri:
 		return TransposeTri{m.(Triangular)}
+	case Transpose:
```

Comment:
Because TransposeTriers are often Transposers as well. We want it to adopt the more specific behavior.